src/main.rs

```diff
@@ -95,24 +157,15 @@ impl DWebBackend {
     }
 }
 
-#[tokio::test]
-async fn basic_test() {
-    let path = "./";
-    let port = 8080;
-
-    let mut d_web_backend = DWebBackend::new(String::from(path), port);
-
-    // Start the backend and wait for SIGINT signal.
-    d_web_backend.start().await.expect("Unable to start");
-    d_web_backend.stop().await.expect("Unable to stop");
-}
-
 #[tokio::main]
 async fn main() -> Result<()> {
-    let path = "./";
+    let path = "./tmp/save_dweb_backend"; // Changed to use a relative temporary directory
```

Comment:
Instead of tmp, lets store in the appropriate `data` folder per OS

Try using this https://github.com/folder/xdg

Set the `subdir` to `save-dweb-backend`

For the tests use this library for the temporary dir: https://docs.rs/tmpdir/latest/tmpdir/